<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelWidget</class>
 <widget class="QWidget" name="ModelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Model Transform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="transformPropWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>262</width>
            <height>144</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Model Mesh info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="meshInfoPropWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>262</width>
            <height>143</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Model material</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="materialPropWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>262</width>
            <height>144</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
